src/tools/genpolicy/tests/generate/testdata/config_map.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: v1
+data:
+  key: YmFyCg==
+kind: ConfigMap
+metadata:
+  name: configmap-sample
+---
+apiVersion: v1
+data:
+  key: YmFyCg==
+kind: ConfigMap
+metadata:
+  name: configmap-sample2
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: one-container-config-map
```

Comment:
Why do we need this pod definition, given that we're only using `config_map.yaml` as side input (and `secret.yaml` fwiw)?

Reply:
Understood. For me, a comment above the (to me) unexpected podSpec would be sufficient.